test/hip-config-test/base/CMakeLists.txt

```
@@ -0,0 +1,38 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(hip-config-test LANGUAGES CXX C)
+
+set(ENABLE_MPI OFF)
+set(ENABLE_FORTRAN OFF)
+set(ENABLE_HIP ON)
+
+# Load BLT
+include(${BLT_SOURCE_DIR}/SetupBLT.cmake)
+
+# Add third-party library setup files to the project's installation
+# directory.
+blt_install_tpl_setups(DESTINATION lib/cmake/${PROJECT_NAME})
+
+# Add the base library with its single source file and header file.
+blt_add_library(
+  NAME hip-config-test
+  SOURCES hip.cpp
+  HEADERS hip.hpp
+  DEPENDS_ON blt_hip blt_hip_runtime)
```

```suggestion
  DEPENDS_ON blt::hip blt::hip_runtime)
```

They technically are the same target but we dont really advertise the _ versions

---

Interesting--I didn't know that